packages/config/tests/cwd/snapshots/tests.js.md

```diff
@@ -20,11 +20,18 @@ Generated by [AVA](https://ava.li).
 
 > Snapshot 1
 
-    `{␊
+    `␊
+    Warning: the "publish" directory was not set and will default to the repository root directory.␊
+    To publish the root directory, please set the "publish" directory to "/"␊
+    To publish the "base" directory instead, please set the "publish" directory to "packages/config"␊
+    ␊
+    ␊
+    {␊
       "branch": "branch",␊
-      "buildDir": "",␊
+      "buildDir": "packages/config",␊
```

Comment:
This is an existing test that runs from `cwd`. It now correctly detects the `config` package as the base directory